packages/transaction-controller/package.json

```diff
@@ -86,7 +86,7 @@
   },
   "peerDependencies": {
     "@babel/runtime": "^7.23.9",
-    "@metamask/accounts-controller": "^17.0.0",
+    "@metamask/accounts-controller": "^17.1.0",
```

Comment:
Do we need this? It may have been bumped by accident:

``` suggestion
    "@metamask/accounts-controller": "^17.0.0",
```

Reply:
Yes, the lint would complain without it

Reply:
Hmm thats odd, its working. I've reverted the change.